.travis.yml

```diff
@@ -35,6 +35,7 @@ before_install:
 before_script:
   - if [[ $PREFER_LOWEST != 1 ]]; then composer install --no-interaction; fi;
   - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-lowest --no-interaction --prefer-dist --prefer-stable; fi;
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer require --dev dereuromark/composer-prefer-lowest; fi
```

Comment:
Is there a way to pin the version?

Reply:
No need to.

Reply:
The dependencies of a build should be idempotent

Reply:
This is my tool, I control it.